### PR TITLE
Add support to automatically create packages/installers with CPack

### PR DESCRIPTION
**Pull request by DefaultUser**
CPack allows easy creation of packages for certain Linux distributions and installers for Windows/macOS.
Note: I only tested this on raspbian

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,15 @@ cmake ..
...
 
+### Creating Packages/Installers using CMake/CPack
+
+Packages for certain Linux distributions can be automatically created using `cpack`. After compilation run:
+```
+cd build
+cpack -G <generator> -D CPACK_PACKAGE_CONTACT="<your contact>"
+```
+For more information see the [CPack documentation](https://cmake.org/cmake/help/latest/manual/cpack.1.html).
+
```
Comment (thestr4ng3r) [inline README.md:60]: I don't think it should be in the readme before I haven't tested it fully yet and can be "officially supported". The changes themselves lgtm though.

**Comment by thestr4ng3r** [comment]
closing because of no response

**Comment by DefaultUser** [comment]
@thestr4ng3r I didn't find the time to answer before.
After your comment I expected you to keep this PR open until you've tested it yourself

Completely removing the section from the README will probably just mean that it will be forgotten, so I'd rather mark it as "Experimental"

It seems like I can't re-open this PR, should I open a new one?

**Comment by thestr4ng3r** [comment]
> Completely removing the section from the README will probably just mean that it will be forgotten, so I'd rather mark it as "Experimental"

I can't say yet if I will keep this and support it in the future. This is why it should not be in the README at all.

> It seems like I can't re-open this PR, should I open a new one?

I've reopened. Feel free to keep using this one.

**Comment by DefaultUser** [comment]
OK, I've removed the section from the README (even though I don't understand why removing am experimental feature in the future would be a problem).

Regarding maintainability: CPack is pretty much "free" when using CMake. I don't expect that there will be many changes needed to keep this feature alive.

**Comment by thestr4ng3r** [comment]
Thanks!

**Pull request merged**